i18n-helpers/src/lib.rs

```diff
@@ -462,7 +462,7 @@ fn heuristic_codeblock<'a>(
         _ => true,
     };
 
-    if is_translate {
+    Ok(if is_translate {
```

Comment:
Nit: wrapping the expression in `Ok` looks to me a bit harder to read but maybe this is the way to do it in rust (haven't done much rust yet). Will this be more readable if you store the result into two variables - `events` and `ctx`, then at the end of the function do `Ok(events, ctx)`?

Reply:
Sure, done! I've written even less rust, so I have no idea what the proper "rust style" would be (or if it there's a preferred way in the first place). But your suggestion looks more readable.